src/js/modules/articles/actionTypes.js

```diff
@@ -1,3 +1,6 @@
 export const CREATE_ARTICLE = 'core/CREATE_ARTICLE'
 export const SAVE_SELECTED_ARTICLES = 'core/SAVE_SELECTED_ARTICLES'
-export const DELETE_SELECTED_ARTICLES = 'core/DELETE_SELECTED_ARTICLES'
\ No newline at end of file
+export const DELETE_SELECTED_ARTICLES = 'core/DELETE_SELECTED_ARTICLES'
+export const FETCH_ARTICLES_REQUESTED = "forms/FETCH_ARTICLES_REQUESTED";
+export const FETCH_ARTICLES_SUCCEEDED = 'forms/FETCH_ARTICLES_SUCCEEDED';
+export const FETCH_ARTICLES_FAILED = 'forms/FETCH_ARTICLES_FAILED'
```

Comment:
The basic idea is FETCH actions should be in module, cause it's consistent with the module features, while CREATE actions should be in the forms module because it's a form feature